user-service/routers/router.go

```diff
@@ -9,8 +9,9 @@ func HandleRequests(h *handlers.UserHandler) *mux.Router {
 	r := mux.NewRouter()
 
 	r.HandleFunc("/register", h.HandleRegistration).Methods("POST")
-
+	r.HandleFunc("/login", h.HandleLogin).Methods("POST")
 	// the verify endpoint is defined as a get so it can redirect when link click happens,
 	r.HandleFunc("/verify", h.HandleAccountVerification).Methods("GET", "POST")
+	r.HandleFunc("/login/verify-otp", h.HandleVerifyLoginOtp).Methods("POST")
```

Comment:
For organizing purposes swap the order of handlers:


```suggestion
	r.HandleFunc("/register", h.HandleRegistration).Methods("POST")

	r.HandleFunc("/login", h.HandleLogin).Methods("POST")
	r.HandleFunc("/login/verify-otp", h.HandleVerifyLoginOtp).Methods("POST")

	// the verify endpoint is defined as a get so it can redirect when link click happens,
	r.HandleFunc("/verify", h.HandleAccountVerification).Methods("GET", "POST")
```